cylc/flow/cfgspec/workflow.py

```diff
@@ -785,9 +785,22 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     :ref:`SequentialTasks`.
             ''')
 
+        Conf('sequential xtriggers default', VDR.V_BOOLEAN, False, desc='''
```

Comment:
Maybe this item could be more descriptive? Something like?:
```suggestion
        Conf('spawn from xtriggers sequentially', VDR.V_BOOLEAN, False, desc='''
```
(and I'm not sure "default" is needed in the name, as opposed to the description) 

(obviously changing this would have some knock-on effects in the code)